### PR TITLE
Boxengine and Title tests

### PR DESCRIPTION
**Pull request by blink1073**
```
----------------|----------|----------|----------|----------|----------------|
File            |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------------|----------|----------|----------|----------|----------------|
 ui/            |      100 |      100 |      100 |      100 |                |
  boxengine.js  |      100 |      100 |      100 |      100 |                |
  title.js      |      100 |      100 |      100 |      100 |                |
----------------|----------|----------|----------|----------|----------------|
All files       |    99.05 |    93.94 |    98.87 |    99.34 |                |
----------------|----------|----------|----------|----------|----------------|

### CODE REVIEW

File: test/src/ui/title.spec.ts [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+        expect(title.label).to.be('');
+      });
+
+      it('should be writable', () => {
+        let title = new Title();
+        title.label = 'foo';
+        expect(title.label).to.be('foo');
+      });
+
+      it('should emit the changed signal', () => {
```
Comment (sccolbert) [inline test/src/ui/title.spec.ts]: ... when the value changes

File: test/src/ui/title.spec.ts [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+        title.changed.connect((sender, arg) => {
+          expect(sender).to.be(title);
+          expect(arg).to.be(void 0);
+          called = true;
+        });
+        title.label = 'baz';
+        expect(called).to.be(true);
+      });
+
+      it('should not emit the changed signal', () => {
```
Comment (sccolbert) [inline test/src/ui/title.spec.ts]: ... when the value does not change

File: test/src/ui/title.spec.ts (hunk 1 above)
Comment (sccolbert) [inline test/src/ui/title.spec.ts]: this also applies to all other get/setters in this file

File: test/src/ui/title.spec.ts (hunk 2 above)
Comment (sccolbert) [inline test/src/ui/title.spec.ts]: this also applies to all other similar tests in this file

File: test/src/ui/boxengine.spec.ts [hunk 3]
```diff
@@ -0,0 +1,346 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2016, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
```
Comment (sccolbert) [inline test/src/ui/boxengine.spec.ts]: 'use strict' is no longer required as of TS 1.8 - it gets added automatically by the compiler

File: test/src/ui/title.spec.ts [hunk 4]
```diff
@@ -0,0 +1,313 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2016, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
```
Comment (sccolbert) [inline test/src/ui/title.spec.ts]: 'use strict' is no longer required as of TS 1.8 - it gets added automatically by the compiler

**Comment by blink1073** [comment]
Comments addressed.

**Pull request merged**